website_sale_require_login/controllers/main.py

```diff
@@ -10,3 +10,7 @@ class RequireLoginToCheckout(WebsiteSale):
     @http.route(auth="user")
     def checkout(self, **post):
         return super().checkout(**post)
+
+    @http.route(auth="user")
+    def address(self, **post):
```

Comment:
Not really sure about this, although it's true that having the customer filling the details without an account will make them work twice...